spec/index.html

```diff
@@ -504,6 +504,43 @@ <h3>RDF Documents and Syntaxes</h3>
       effect on the convenience of working with the <a>RDF document</a>,
       they are not significant for its meaning.</p>
   </section>
+
+  <section id="section-version-announcement">
+    <h3>RDF Version Announcement</h3>
+
+    <p>To allow RDF parsers to error or warn about unsupported RDF versions as early as possible,
+      RDF serialization formats are expected to allow a version to be specified,
+      via either the HTTP Content-Type header,
+      a version announcement in a format-specific syntax,
+      or both.</p>
+
+    <p>If the version is announced in both the Content-Type header and the syntax,
+      they are expected to be equal.</p>
```

Comment:
```suggestion
<p>When the version is indicated both in a media-type parameter and in syntax,
	  they are expected to be the same.
	  If they differ,
	  parsers use the version from the media-type parameter and might emit a warning about the mismatch.</p>
```

Reply:
If I upload a Turtle file with a version indicator to to some web server and the server then serves the file with a mismatching version in the media type, isn't it more likely that the web server is wrong?

Reply:
Perhaps in a general sense, but that kind of roundtripping interoperability isn't typically checked (AFAIK). What is checked is server-client interoperability, or more broadly, sender-receiver. In this case, the server's representation metadata is authoritative and doesn't reflect the original version (regardless of how it was obtained).